sys/net/gnrc/routing/rpl/of0.c

```diff
@@ -87,9 +87,57 @@ int parent_cmp(gnrc_rpl_parent_t *parent1, gnrc_rpl_parent_t *parent2)
     return 0;
 }
 
-/* Not used yet */
-gnrc_rpl_dodag_t *which_dodag(gnrc_rpl_dodag_t *d1, gnrc_rpl_dodag_t *d2)
+int which_dodag(gnrc_rpl_dodag_t *d1, gnrc_rpl_dio_t *dio)
 {
```

Comment:
If we wanted to compare two `gnrc_rpl_dodag_t`s, we'd first need to allocate the whole dodag structure based on the DIO, and even create a new RPL instance too because we only support one DODAG per instance. 
I don't think this makes sense, given that in practice we only ever compare the info from `DIO`s with the current DODAG.